lib/msal-browser/test/interaction_handler/SilentHandler.spec.ts

```diff
@@ -389,5 +397,81 @@ describe("SilentHandler.ts Unit Tests", () => {
                 };
             }, 500);
         });
+
+        it("Throws hash empty error", (done) => {
+            const iframe = {
+                contentWindow: {
+                    location: {
+                        href: "about:blank",
+                        hash: "",
+                    },
+                },
+            };
+
+            const silentHandler = new SilentHandler(
+                authCodeModule,
+                browserStorage,
+                defaultTokenRequest,
+                browserRequestLogger,
+                {
+                    navigateFrameWait: DEFAULT_IFRAME_TIMEOUT_MS,
+                    pollIntervalMilliseconds: DEFAULT_POLL_INTERVAL_MS,
+                },
+                performanceClient
+            );
+            // @ts-ignore
+            silentHandler.monitorIframeForHash(iframe, 1000).catch((e) => {
+                expect(e).toBeInstanceOf(BrowserAuthError);
+                expect(e).toMatchObject(
+                    BrowserAuthError.createEmptyHashError()
+                );
+                done();
+            });
+
+            setTimeout(() => {
+                iframe.contentWindow.location = {
+                    href: "http://localhost/",
+                    hash: "",
+                };
+            }, 500);
+        });
+
+        it("Throws hashDoesNotContainKnownProperties error", (done) => {
+            const iframe = {
+                contentWindow: {
+                    location: {
+                        href: "about:blank",
+                        hash: "",
+                    },
+                },
+            };
+
+            const silentHandler = new SilentHandler(
+                authCodeModule,
+                browserStorage,
+                defaultTokenRequest,
+                browserRequestLogger,
+                {
+                    navigateFrameWait: DEFAULT_IFRAME_TIMEOUT_MS,
+                    pollIntervalMilliseconds: DEFAULT_POLL_INTERVAL_MS,
+                },
+                performanceClient
+            );
+            // @ts-ignore
+            silentHandler.monitorIframeForHash(iframe, 1000).catch((e) => {
```

Comment:
Would be great to use constants for the 1000 here and the 500 on line 474 just below.